app/templates/partials/question.html

```diff
@@ -24,13 +24,55 @@ <h3 class="question__subtitle mars">{{question_subtitle}}</h3>
 {% endset %}
 
 {% set question_answers %}
-  <div class="question__answers">
-    {%- for answer in question.answers -%}
-      {% include theme('partials/answer.html') %}
-    {%- endfor -%}
+ <div class="question__answers">
+   {%- if question.schema_item.type == 'RepeatingAnswer' %}
+     {%- for answer in question.answers %}
+       <div class="question__answer js-household-person">
+         <h3 class="neptune">Person <span class="js-household-loopindex">{{ loop.index }}</span>
+           <small>(
+           {%- if loop.index == 1 -%}
+              <span class="js-household-action">{{ _('You') }}</span>
+           {%- else -%}
+           <button class="btn btn--link"
+              formaction="{{ url_for('action.remove_answer',
```

Comment:
Not sure how I feel able this, maybe we should just hard code the url with the dynamic parts substituted or create a add/remove variable in the context